src/schema/value.rs

```diff
@@ -366,6 +389,10 @@ mod binary_serialize {
                     serde_json::to_writer(writer, &map)?;
                     Ok(())
                 }
+                Value::Ip(ref ip) => {
+                    IP_CODE.serialize(writer)?;
+                    ip.to_string().serialize(writer) // TODO Check best format
```

Comment:
I'd rather use IPv6 all the time and encode stuff over 16 bytes.

Reply:
I did some tests and u128 is probably better most of the time